src/app/components/SubjectHeading/SubjectHeading.jsx

```diff
@@ -254,18 +255,19 @@ class SubjectHeading extends React.Component {
     const isMain = (pathname + search).includes(uuid);
     // changes to HTML structure here will need to be replicated in ./SubjectHeadingTableHeader
 
+
     return (
       <React.Fragment>
         <tr
           className={`
             subjectHeadingRow
-            ${(open || children || isMain) ? 'openSubjectHeading' : ''}
-            ${(indentation || 0) === 0 ? 'topLevel' : ''}
-            ${(indentation || 0) !== 0 ? 'nestedSubjectHeading' : ''}
+            ${open && children ? 'openSubjectHeading' : ''}
+            ${indentation === 0 ? 'topLevel' : ''}
```

Comment:
Are we sure that indentation is always a number? I seem to remember that it could be undefined or nil in some cases.

Reply:
Yes. It has a default prop of 0.

Reply:
Also, this `topLevel` class is never actually used it seems. But I don't mind having it there incase we eventually need to apply styling just to the top level headings.

Reply:
When would `narrower.length` be falsey? It looks like it is initialized to an empty array.

Reply:
If it is an empty array. It is initialized with ` narrower: (children || [])`

Reply:
Got it. So if a subject heading is open but has no children, do we want it to not receive `openSubjectHeading` styling?

Reply:
I thought I left a comment but I guess it got lost. The only styling from `openSubjectHeading` is the removal of the bottom border. `.selected` does the bolding. `.selected` is determined based on the `onMainPath` property I added to the headings.

Reply:
Oh, here is that comment: https://github.com/NYPL-discovery/discovery-front-end/pull/1245#discussion_r405696507